Client/Frontend/Reader/View/ReaderModeBarView.swift

```diff
@@ -102,7 +108,7 @@ class ReaderModeBarView: UIView, AlphaDimmable, TopBottomInterchangeable, Search
         guard let context = UIGraphicsGetCurrentContext() else { return }
```

Comment:
Safe unwrap of the color
```suggestion
        guard let context = UIGraphicsGetCurrentContext(),
              let contextStrokeColor = contextStrokeColor else { return }
```
The result of this change is that if the color doesn't get initialized we won't draw the line which is preferable than crashing